gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -368,7 +368,7 @@
 
 		for (final User u : users) {
 			gbUsers.add(new GbUser(u, getStudentNumber(u, site))
-							.setSections(userSections.getOrDefault(u.getEid(), Collections.emptyList())));
+							.setSections(userSections.getOrDefault(u.getId(), Collections.emptyList())));
```

Comment:
Are you sure this works with sections from inside Sakai, too? So, a mixture of cms provided and Sakai provided?

Reply:
yes works for manually-created sections and CM API-created sections

Reply:
🚀